subgraph-kovan-staging.yaml

```diff
@@ -54,7 +54,7 @@ dataSources:
     source:
       address: "0xCcf827238991FC3D5A51067968501A38624F313d"
       abi: Shelf
-      startBlock: 18594066
+      startBlock: 18282946
```

Comment:
Why do we need to start the Shelf from an earlier block? Shouldn’t this be for the proxy registry https://github.com/centrifuge/tinlake-subgraph/pull/30/files#diff-e6bbf6c2e388037f1ef13fedf3ba0417R36 ?

Reply:
haha, of course the proxy registry. I edited the wrong block. Thx